Source/WebKit/UIProcess/WebPageProxy.cpp

```diff
@@ -11278,8 +11280,6 @@ void WebPageProxy::resetStateAfterProcessExited(ProcessTerminationReason termina
         protectedPageLoadState->reset(transaction);
     }
 
-    updatePlayingMediaDidChange(MediaProducer::IsNotPlaying);
```

Comment:
This seemed unnecessary since we were already calling resetState in this function which also calls updatePlayingMediaDidChange.